BitSliceIndexing/bsi.go

```diff
@@ -474,7 +585,11 @@ func (b *BSI) ClearValues(foundSet *roaring.Bitmap) {
 	wg.Wait()
 }
 
-// NewBSIRetainSet
+/*
+ * NewBSIRetainSet
+ *
+ * Construct a new BSI from a clone of existing BSI, retain only values contained in foundSet
+ */
 func (b *BSI) NewBSIRetainSet(foundSet *roaring.Bitmap) *BSI {
```

Comment:
Comment on NewBSIRetainSet does not follow the expected format.

Reply:
Fixed.